Networking/Services/SessionManager.swift

```diff
@@ -112,14 +111,18 @@ public protocol SessionManagerType {
  */
 final public class SessionManager: SessionManagerType {
 
-    fileprivate let _keychain: KeychainSwift
+    fileprivate let _keychainService: KeychainServiceType
     fileprivate var _currentUserFetcher: CurrentUserFetcherType?
     
     fileprivate let _sessionToken = MutableProperty<String?>(.none)
     fileprivate let _currentUser = MutableProperty<AuthenticableUser?>(.none)
     
-    public init(keychain: KeychainSwift = KeychainSwift()) {
-        _keychain = keychain
+    convenience public init() {
```

Comment:
Right, but know we wouldn't use `KeychainSwift`, but the public protocol `KeychainServiceType`. I think that init should be public, since is the general one. Then by having a default, you could make this init public also, but the other must be public for me at least.